platform/security/sts/security-sts-realm/src/main/java/ddf/security/realm/sts/SamlRealm.java

```diff
@@ -0,0 +1,187 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.security.realm.sts;
```

Comment:
❗️ ❓ Why is the `SamlRealm` related to the STS? Isn't this PR removing the STS?

Reply:
The server is removed. I'm not renaming things or removing packages in this PR.